### PR TITLE
A couple more patches to fix two more tests

### PR DESCRIPTION
**Pull request by abradle**

### CODE REVIEW

**Comment by abradle** [comment]
Feel free to wait - I'll hopefully fix a few more over the next half hour.

File: biojava-structure/src/main/java/org/biojava/nbio/structure/align/multiple/util/MultipleAlignmentWriter.java [hunk 1]
```diff
@@ -175,7 +175,8 @@ public static String toAlignedResidues(MultipleAlignment multAln) {
 								.toString());
 						residueGroup.append('\t');
 						residueGroup.append(atom.getGroup().getChain()
-								.getChainID());
+				// ABradley - I'm assuming Auth Id's here 04/05/16
```
Comment (abradle) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/align/multiple/util/MultipleAlignmentWriter.java:178]: Just checking @andreasprlic @josemduarte this is supposed to be auth?

Comment (andreasprlic) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/align/multiple/util/MultipleAlignmentWriter.java:178]: Yes I think so

File: biojava-structure/src/main/java/org/biojava/nbio/structure/StructureImpl.java [hunk 2]
```diff
@@ -249,7 +249,7 @@ public Chain findChain(String chainId, int modelnr) throws StructureException {
 
 		// iterate over all chains.
 		for (Chain c : chains) {
-			if (c.getChainID().equals(chainId)) {
+			if (c.getName().equals(chainId)) {
```
Comment (andreasprlic) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/StructureImpl.java:252]: Can you rename the argument to authId? I tried to make this consistent wherever I found this.

**Comment by abradle** [comment]
Ok - that's me done for now. 

There's four failures and an empty bounding box error -  all domain/bioassembly related. I believe this is @josemduarte s specialist subject.

**Comment by josemduarte** [comment]
Alright I'll have a look at those

**Pull request merged**